platformio.ini

```diff
@@ -6,14 +6,15 @@ extra_scripts =
   pre:tools/constants_generator.py
```

Comment:
Do we want to add the Jacobean generator as a prebuild script as well?

Reply:
No, because the output needs to be manually edited and checked before its used.

Reply:
Yep, wasn't aware there was a manual step to it as well.